Sources/TensorFlow/Layers/Core.swift

```diff
@@ -138,6 +138,11 @@ public struct Reshape<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
 /// `Dense` implements the operation `activation(matmul(input, weight) + bias)`, where `weight` is
 /// a weight matrix, `bias` is a bias vector, and `activation` is an element-wise activation
 /// function.
+///
+/// This layer also supports 3-dimensional weight tensors with 2-dimensional bias matrices. In this 
```

Comment:
I think we've been using "n-D" in most API documentation. If so, I suggest changing this. Otherwise never mind.
```suggestion
/// This layer also supports 3-D weight tensors with 2-D bias matrices. In this 
```